zerver/tests/test_signup.py

```diff
@@ -2448,7 +2448,7 @@ def test_create_realm_during_free_trial(self) -> None:
         self.assertEqual(result.url, 'http://zuliptest.testserver/upgrade/?onboarding=true')
 
         result = self.client_get(result.url, subdomain=string_id)
-        self.assert_in_success_response(["Not ready to start your trial?"], result)
+        self.assert_in_success_response(["How do you want to get started?"], result)
```

Comment:
I'm not a big fan of the phrase "How do you want to get started" -- maybe "Choose a plan for your new organization"?